spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDLKeras.scala

```diff
@@ -56,11 +57,34 @@ class PythonBigDLKeras[T: ClassTag](implicit ev: TensorNumeric[T]) extends Pytho
     }
   }
 
+  def toScalaArray(list: JList[Int]): Array[Int] = {
+    if (list == null) {
+      null
+    } else {
+      list.asScala.toArray
+    }
+  }
+
   def createKerasInputLayer(
     inputShape: JList[Int] = null): Input[T] = {
     InputLayer(inputShape = toScalaShape(inputShape))
   }
 
+  def getOutputShape(module: Container[Activity, Activity, T]): JList[Int] = {
```

Comment:
There's possible that `outputshape` is MultiShape.

Reply:
In what case outputShape is a MultiShape?